luatest/server.lua

```diff
@@ -333,10 +446,14 @@ function Server:start(opts)
     })
 
     local wait_until_ready
-    if self.coverage_report then
-        wait_until_ready = self.original_command == DEFAULT_INSTANCE
+    if self.config_file then
+        wait_until_ready = self.net_box_uri ~= nil
```

Comment:
What is the idea behind the `self.net_box_uri` check?

Reply:
If `self.net_box_uri` is `nil`, we will be unable to check the server is ready because we need URI to connect to the server. So `wait_until_ready` should be `false` in this case.

Reply:
I've replaced this statement with `wait_until_ready = true` and run all the test in tarantool and tarantool-ee that use this helper. No tests were failed. OK, let's assume that it is just to don't fail in some theoretical case, when we don't set the URI.